histomicstk/cli/PodoSighter_cnn/PodoSighter_cnn.py

```diff
@@ -3,17 +3,11 @@
 import sys
 
 def main(args2):         
-    
-    #cmd = "python3 ../PodoSighter_pix2pix_folder/Complete_Pix2pix_Prediction.py -A0 '{}' -A1 '{}' -A2 '{}' -A3 '{}' -A4 '{}' -A5 '{}' -A6 {} -A7 {} -A8 {} -A9 '{}' -A10 '{}' -A11 {}".format(args2.inputFolder, args2.inputImageFile,args2.inputAnnotationFile1, args2.TrainedGeneratorModel ,args2.TrainedDiscriminatorModel,args2.outputAnnotationFile1, args2.PASnucleiThreshold,args2.gauss_filt_size, args2.Disc_size ,args2.species,args2.stain,args2.gpu_id)
-    
+     
     
     sys.path.append("../PodoSighter_cnn_folder/")
-    
-
-    #cmd = "python3 ../PodoSighter_cnn_folder/Complete_CNN_Prediction.py -A0 '{}' -A1 '{}' -A2 '{}' -A3 '{}' -A4 '{}' -A5 '{}' -A6 '{}' -A7 '{}' -A8 '{}' -A9 {} -A10 {} -A11 {} -A12 {} -A13 {} -A14 {} -A15 '{}'".format(args2.inputFolder, args2.inputImageFile,args2.inputAnnotationFile1, args2.Model,args2.Modelchkpt,args2.Modelidx, args2.species,args2.stain, args2.outputAnnotationFile1, args2.PASnucleiThreshold,args2.gauss_filt_size, args2.Disc_size,args2.resolut,args2.sz_thre,args2.watershed_thre,args2.outputAnnotationFile2)
-    
-    #cmd = "python3 ../PodoSighter_cnn_folder/Complete_CNN_Prediction.py -A0 '{}' -A1 '{}' -A2 '{}' -A3 '{}' -A4 '{}' -A5 '{}' -A6 '{}' -A7 {} -A8 {} -A9 {} -A10 {} -A11 {} -A12 {} -A13 '{}' -A14 '{}'".format(args2.inputFolder, args2.inputImageFile,args2.inputAnnotationFile1, args2.Model,args2.Modelchkpt,args2.Modelidx, args2.species, args2.PASnucleiThreshold,args2.gauss_filt_size, args2.Disc_size,args2.resolution,args2.size_thre,args2.watershed_thre,args2.outputAnnotationFile2,args2.outputAnnotationFile1)
-    cmd = "python3 ../PodoSighter_cnn_folder/Complete_CNN_Prediction.py -A0 '{}' -A1 '{}' -A2 '{}' -A3 '{}' -A4 '{}' -A5 '{}' -A6 '{}' -A7 {} -A8 {} -A9 {} -A10 {} -A11 {} -A12 {} -A13 '{}' -A14 '{}' -A15 {} -A16 '{}'".format(args2.inputFolder, args2.inputImageFile,args2.inputAnnotationFile1, args2.Model,args2.Modelchkpt,args2.Modelidx, args2.species, args2.PASnucleiThreshold,args2.gauss_filt_size, args2.Disc_size,args2.resolution,args2.size_thre,args2.watershed_thre,args2.outputAnnotationFile2,args2.outputAnnotationFile1,args2.tissue_thickness,args2.csvfilename)
+        
+    cmd = "python3 ../PodoSighter_cnn_folder/Complete_CNN_Prediction.py -A0 '{}' -A1 '{}' -A2 '{}' -A3 '{}' -A4 '{}' -A5 '{}' -A6 '{}' -A7 {} -A8 {} -A9 {} -A10 {} -A11 {} -A12 {} -A13 '{}' -A14 '{}' -A15 {} -A16 '{}' -A17 '{}' -A18 '{}'".format(args2.inputFolder, args2.inputImageFile,args2.inputAnnotationFile1, args2.Model,args2.Modelchkpt,args2.Modelidx, args2.species, args2.PASnucleiThreshold,args2.gauss_filt_size, args2.Disc_size,args2.resolution,args2.size_thre,args2.watershed_thre,args2.outputAnnotationFile2,args2.outputAnnotationFile1,args2.tissue_thickness,args2.csvfilename,args2.girderApiUrl,args2.girderToken)
 

```

Comment:
Do we pass `args2.species` to `Complete_CNN_Prediction.py` file?

Reply:
Yes. It is an input parameter